view.js

```diff
@@ -66,15 +66,15 @@ export function setCompleteAllCheckbox(checked) {
 
 export function updateFilterButtons(route) {
 	qs('.filters .selected').className = '';
-	qs(`.filters [href="#/${route}"]`).className = 'selected';
+	qs(`.filters [href='#/${route}']`).className = 'selected';
```

Comment:
quotes in the xml are ok. again, I prefer minimal change

Reply:
Just wondering, why have inconsistent quotes? Why not all single or all double?